devel_entrypoint.sh

```diff
@@ -1,4 +1,5 @@
-#!/bin/env bash
+#!/usr/bin/env bash
+
 set -e
-source /opt/ros/$ROS_DISTRO/setup.bash
+source "/ros_entrypoint.sh"
```

Comment:
Where is this file?

Reply:
inside the container, the base image is responsible for it

Reply:
I think there is no need to override the entry point of the base image (check [this](https://stackoverflow.com/questions/43237183/dockerfile-how-use-cmd-or-entrypoint-from-base-image))

Reply:
fixed 8fbca5f